Add in a page for companies supporting file formats. (#403)
hyperspy/rosettasciio: doc/supported_formats/supporting_your_file_format.rst

.. _supporting_your_file_format:

Supporting Your File Format
===========================

> Many companies have decided to support their own file formats and we strongly support and encourage this.

Reviewer: ```suggestion
**Many companies have decided to develop and support plugins for their own file formats or the file formats associated with their products and we strongly support and encourage this commitment.**
```

> This helps ensure that the data is being read correctly and often allow for more complete file reading.

Reviewer: ```suggestion
This engagement helps ensure that the data is being read correctly and often allows for more comprehensive file reading and metadata parsing. For companies, it has the advantage that customers/users get easy access to the vast possibilities of data analysis using various scientific python packages.
```

> If you are a user of RosettaSciIO or HyperSpy and have a file format that you would like to support, please consider

Reviewer: ```suggestion
If you are a user of RosettaSciIO or HyperSpy and have a file format that you would like to have support for, please consider
```

contacting the company to ask them to support their file format in RosettaSciIO.

The following is a list of companies that are currently or have previously supported the reading/writing

> of their file formats in RosettaSciIO.  We'd like to thank them for their support and continued commitments to open

Reviewer: ```suggestion
of their file formats in RosettaSciIO.  We would like to thank them for their support and continued commitment to open
```

file formats.

.. note:: We are always looking for more companies to support their file formats.

>    If you are a company that would like to support your file format or have previously supported it,

Reviewer: ```suggestion
   If you represent a company that would like to support their file format or have previously supported it,
```
